packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -1,26 +1,18 @@
 <template>
   <va-input-wrapper
```

Comment:
Can't type any text in the all inputs

Reply:
![image](https://github.com/epicmaxco/vuestic-ui/assets/35342240/f24e778e-eac8-4135-a916-610a03db905b)
fix vertical align please